integrations/amazon_bedrock/tests/test_text_embedder.py

```diff
@@ -84,67 +84,62 @@ def test_from_dict(self, mock_boto3_session):
         assert embedder.model == "cohere.embed-english-v3"
         assert embedder.kwargs == {"input_type": "search_query"}
 
+    def test_init_invalid_model(self):
```

Comment:
the reviewer can ignore this file:
I only indented the tests to put them in the test class.